src/main/kotlin/com/sourcegraph/cody/agent/protocol/ProtocolTextDocument.kt

```diff
@@ -3,7 +3,7 @@ package com.sourcegraph.cody.agent.protocol
 import com.intellij.openapi.vfs.VirtualFile
 import com.sourcegraph.cody.agent.protocol.util.Rfc3986UriEncoder
 
-class TextDocument
+class ProtocolTextDocument
```

Comment:
Why is this named this way? It looks awkward given it is in a package "protocol" and it has a bunch of sibling types that are TextDocumentFoo.